extractor/extractor.py

```diff
@@ -0,0 +1,223 @@
+import bs4
+from bs4 import Comment
+import re
+from urllib.request import urlopen
+import PIL.Image
+import requests
+from pprint import pprint
+import contents
```

Comment:
Group and sort the imports in PEP8 guidelines.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
It helped a lot! Updated accordingly.